.github/workflows/paralleltest.yml

```diff
@@ -116,15 +132,26 @@ jobs:
           path: ${{ github.workspace }}/aqa-tests/TKG/parallelList.mk
 
   run-aqa-parallel:
+    name: Run AQAvit verification ${{ inputs.system }} - _${{ matrix.suite }}.${{ matrix.test_list }}
     runs-on: ${{ fromJSON(needs.configuration.outputs.agent-config) }}
     needs: [setup-parallel, configuration]
-    name: Run AQAvit verification ${{ inputs.system }} - _${{ matrix.suite }}.${{ matrix.test_list }}
+    timeout-minutes: 1440
     strategy:
       fail-fast: false
       matrix:
         test_list: ${{ fromJSON(needs.configuration.outputs.testLists) }}
         suite: [functional, openjdk, system, perf]
     steps:
+      - name: clean up workspace (non-windows)
+        working-directory: ${{ github.workspace }}
+        shell: bash
+        if: ${{ inputs.system != 'windows' }}
+        run: |
+          ls -la ./
+          rm -rf ./*
+          rm -rf ./.??*
+          ls -la ./
```

Comment:
The `cleanup workspace` step above omits the `ls -la ./` step at the end. Did you mean to include that there, or omit this one?